perf/benchmark/runner/runner.py

```diff
@@ -129,6 +131,13 @@ def serversidecar(self, fortio_cmd):
         return fortio_cmd + "_serveronly " + basestr.format(
             svc=self.server.ip, port=self.ports[self.mode]["port"], size=self.size)
 
+    def clientsidecar(self, fortio_cmd):
+        basestr = "http://{svc}:{port}/echo?size={size}"
+        if self.mode == "grpc":
+            basestr = "-payload-size {size} {svc}:{port}"
+        return fortio_cmd + "_clientonly " + basestr.format(
+            svc=self.server.labels["app"], port=self.ports[self.mode]["direct_port"], size=self.size)
+
```

Comment:
@mandarjog I need your suggestions about the easiest way/command to set up a clientonly mode request. Like we should use server_pod_ip or server_name, which port we should use? How to by pass a server side car to make it as if we do not have server sidecar? Thank you!

Reply:
There may be different ways but I think youcan use excludeInboundPorts/excludeOutboundPorts annotations. For example, the direct_port we use in server sidecar only mode is added to excludeInboundPorts